README.md

```diff
@@ -1,136 +1,89 @@
+| [camel-jbossdatagrid-fuse](camel-jbossdatagrid-fuse/README.md) | Camel, Data Grid, REST | Use the Camel component camel-jbossdatagrid in JBoss Fuse. |
+| [spark](spark/README.md) | Data Grid, Apache Spark | Read and write data to Data Grid and use streaming to react to data changes in real time. |
 
-* Regardless of your level of expertise, we suggest you start with the **helloworld** quickstart. It is the simplest example and is an easy way to prove your server is configured and started correctly.
-* If you are a beginner or new to JBoss, start with the quickstarts labeled **Beginner**, then try those marked as **Intermediate**. When you are comfortable with those, move on to the **Advanced** quickstarts.
-* Some quickstarts are based upon other quickstarts but have expanded capabilities and functionality. If a prerequisite quickstart is listed, be sure to deploy and test it before looking at the expanded version.
+## System Requirements
+You need the following to build and run the quickstarts:
 
+* Java 8.0 (Java SDK 1.8) or later.
+* Maven 3.0 or later. See the [Maven Getting Started Guide](http://maven.apache.org/guides/getting-started/index.html).
+* JBoss EAP server distribution. Available from the [Red Hat Customer Portal](https://access.redhat.com/downloads).
+* RHDG server distribution for _Remote Client-Server Mode_. Available from the [Red Hat Customer Portal](https://access.redhat.com/downloads).
 
-System Requirements
--------------------
+You can also run the quickstarts with [JBoss Developer Studio or Eclipse](#use-jboss-developer-studio-or-eclipse-to-run-the-quickstarts).
 
-To run these quickstarts with the provided build scripts, you need the following:
+### Configuring Maven
 
-1. Java 1.8, to run JBoss Data Grid, JBoss EAP and Maven. You can choose from the following:
-    * OpenJDK
-    * Oracle Java SE
-    * Oracle JRockit
+If you have not yet done so, you must [Configure Maven](https://github.com/jboss-developer/jboss-developer-shared-resources/blob/master/guides/CONFIGURE_MAVEN.md#configure-maven-to-build-and-deploy-the-quickstarts).
 
-2. Maven 3.0.0 or newer, to build and deploy the examples
-    * If you have not yet installed Maven, see the [Maven Getting Started Guide](http://maven.apache.org/guides/getting-started/index.html) for details.
-    * If you have installed Maven, you can check the version by typing the following in a command prompt:
+If you do not configure Maven, you must pass settings with every Maven command as follows: ` -s QUICKSTART_HOME/settings.xml`
 
-            mvn --version 
+### Setting Environment Variables
+RHDG quickstarts use the following environment variables:
 
-3. The JBoss Data Grid server distribution ZIP.
-    * For information on how to install and run JBoss Data Grid, refer to the product documentation located on the Customer Portal here: <https://access.redhat.com/site/documentation/en-US/JBoss_Data_Grid/>.
+* `JBOSS_HOME` denotes the root JBoss server directory.
```

Comment:
This is confusing, what's the "JBoss server directory" ?

Reply:
@galderz I've updated the description